sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/JsonNodeStorePayload.java

```diff
@@ -40,4 +71,45 @@ public int getResponsePayloadSize() {
     public JsonNode getPayload() {
         return jsonValue;
     }
+
+    private static CharsetDecoder getFallbackCharsetDecoder() {
+        if (StringUtil.isNullOrEmpty(Configs.getCharsetDecoderErrorActionOnMalformedInput())
+         && StringUtil.isNullOrEmpty(Configs.getCharsetDecoderErrorActionOnMalformedInput())) {
```

Comment:
seems this check is redundant

Reply:
Good catch, here is a following PR to change it: https://github.com/Azure/azure-sdk-for-java/pull/42637/files